xbmc/pvr/PVRDatabase.cpp

```diff
@@ -520,7 +531,7 @@ int CPVRDatabase::Get(CPVRChannelGroup &group)
         if (channel)
         {
 #if PVRDB_DEBUGGING
-          CLog::Log(LOGDEBUG, "PVR - %s - channel '%s' loaded from the database", __FUNCTION__, channel->m_strChannelName.c_str());
+          CLog::Log(LOGDEBUG, "CPVRDatabase - %s - channel '%s' loaded from the database", __FUNCTION__, channel->m_strChannelName.c_str());
```

Comment:
please don't mix cosmetics with fixes or features, this makes a review or later a search really hard